enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+only permit and exclude times.
+
+- Service Delivery will reserve the right to disable this strategy for emergency corrective actions.
+- Service Delivery should constrain permit & exclude configurations based on their internal policies. For example, customers may be forced to enable permissive windows which amount to at least 6 hours a month.
+
+### Implementation Details/Notes/Constraints
+
+#### ChangeManagement Stanza
+The change management stanza will be introduced into ClusterVersion and MachineConfigPool (for standalone profiles)
+and HostedCluster and NodePool (for HCP profiles). The structure of the stanza is:
+
+```yaml
+spec:
+  changeManagement:
+    # The active strategy for change management (unless disabled by disabledUntil).
+    strategy: <strategy-name|Disabled|null>
+    
+    # If set to true or a future date, the effective change management strategy is Disabled. Date 
+    # must be RFC3339. 
+    disabledUntil: <bool|date|null>
+    
+    # If set to true or a future date, all strategies other than Disabled are paused. Date 
+    # must be RFC3339. 
+    pausedUntil: <bool|date|null>    
+    
+    # If a strategy needs additional configuration information, it can read a 
+    # key bearing its name in the config stanza.
+    config:
+       <strategy-name>:
+         ...configuration policy for the strategy...
+```
+
+#### MaintenanceSchedule Strategy Configuration
+
+```yaml
+spec:
+  changeManagement:
+    strategy: MaintenanceSchedule
+    config:
+       maintenanceSchedule:
+        # Specifies a reoccurring permissive window. 
+        permit:  
+          # RRULEs (https://www.rfc-editor.org/rfc/rfc5545#section-3.3.10) are commonly used 
+          # for calendar management metadata. Only a subset of the RFC is supported. If
+          # unset, all dates are permitted and only exclude constrains permissive windows.
+          recurrence: <rrule|null>
+          # Given the identification of a date by an RRULE, at what time (relative to timezoneOffset) can the 
+          # permissive window begin. "00:00" if unset.
+          startTime: <time-of-day|null>
+          # Given the identification of a date by an RRULE, at what time (relative to timezoneOffset) should the 
+          # permissive window end. "23:59:59" if unset.
+          endTime: <time-of-day|null>
+        
+        # Excluded date ranges override RRULE selections.
+        exclude:
+        # Dates should be specified in YYYY-MM-DD. Each date is excluded from 00:00<timezoneOffset> for 24 hours.
+        - fromDate: <date>
+          # Non-inclusive until. If null, until defaults to the day after from (meaning a single day exclusion).
+          untilDate: <date|null>
+
+        # Specifies an RFC3339 style timezone offset to be applied across their datetime selections.
+        # "-07:00" indicates negative 7 hour offset from UTC. "+03:00" indicates positive 3 hour offset. If not set, defaults to "+00:00" (UTC). 
+        timezoneOffset: <null|str>
```

Comment:
In my opinion the use of timezone will complicate things and make it confusing for those who will look at this configuration or manage it.  I recommend removing the ability to set a timezone offset and stick with UTC in all scenarios.